# SOME DESCRIPTIVE TITLE.
# Copyright (C) YEAR ORGANIZATION
# FIRST AUTHOR <EMAIL@ADDRESS>, YEAR.
#
msgid ""
msgstr ""
"Project-Id-Version: \n"
"POT-Creation-Date: 2016-07-17 13:54+0200\n"
"PO-Revision-Date: 2016-07-17 13:55+0200\n"
"Last-Translator: \n"
"Language-Team: \n"
"Language: de_DE\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"
"Generated-By: pygettext.py 1.5\n"
"X-Generator: Poedit 1.8.8\n"
"Plural-Forms: nplurals=2; plural=(n != 1);\n"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:63
msgid "Update failed!"
msgstr "Update fehlgeschlagen!"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:63
#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:4131
#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:4141
msgid "Could not connect to %s"
msgstr "Verbindung zu %s fehlgeschlagen"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:258
#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:260
#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:262
msgid "Operating System not supported: %s"
msgstr "Betriebssystem wird nicht unterstützt: %s"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:277
msgid "Copy old appdata to new folder failed!"
msgstr "Kopiere \"appdata\" in neues Verzeichnis fehlgeschlagen!"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:277
msgid "Error: def win_pre1_check_app_dir()"
msgstr "Fehler: def win_pre1_check_app_dir()"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:288
msgid "Could not create app_dir: %s"
msgstr "Konnte App Verzeichnis nicht anlegen: %s"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:315
msgid "Select User-ID failed!"
msgstr "User-ID Auswahl fehlgeschlagen!"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:380
#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:383
#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:386
#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:389
#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:392
msgid "Error: '%s' not found!"
msgstr "Fehler: '%s' nicht gefunden!"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:409
msgid ""
"Could not remove lock file.\n"
"File itself locked because another oVPN Client instance running?"
msgstr ""
"Sperr Datei konnte nicht entfernt werden.\n"
"Sperr Datei selbst ist gesperrt. Läuft eine weitere oVPN Client Instanz?"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:454
msgid ""
"Creating API-DIRS\n"
"%s \n"
"%s \n"
"%s \n"
"%s \n"
"%s failed!"
msgstr ""
"Anlegen von API-DIRS\n"
"%s \n"
"%s \n"
"%s \n"
"%s \n"
"%s fehlgeschlagen!"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:456
msgid "Creating config Folders failed"
msgstr "Erzeugen der Konfigurationsverzeichnisse fehlgeschlage"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:683
msgid "Error: def read_options_file"
msgstr "Fehler: def read_options_file"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:683
msgid "Read config file failed!"
msgstr "Lesen der Konfigurationsdatei fehlgeschlagen!"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:816
msgid ""
"Error reading your Interfaces from netsh.exe. Contact support@ovpn.to with "
"Error-ID:\n"
"ADAPTERS[1]=(%s)\n"
msgstr ""
"Konnte die Interfaces mit netsh.exe nicht auslesen. Kontaktiere support@ovpn."
"to mit Error-ID:\n"
"ADAPTERS[1]=(%s)\n"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:851
msgid ""
"Could not read your Network Interfaces! Please install OpenVPN or check if "
"your TAP-Adapter is really enabled and driver installed."
msgstr ""
"Konnte Netzwerk Adapter nicht auslesen! Bitte installiere openVPN oder "
"überprüfe ob der TAP Adapter installiert und aktiviert ist."

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:882
msgid "No OpenVPN TAP-Windows Adapter found!"
msgstr "Kein openVPN TAP Adapter gefunden!"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:896
msgid ""
"Multiple TAPs found!\n"
"\n"
"Please select your TAP Adapter!"
msgstr ""
"Mehrere TAP Adapter gefunden!\n"
"\n"
"Bitte den zu benutzenden Adapter auswählen!"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:912
msgid ""
"No OpenVPN TAP-Adapter found!\n"
"Please install openVPN!\n"
"URL1: %s\n"
"URL2: %s"
msgstr ""
"Kein openVPN TAP Adapter gefunden!\n"
"Bitte openVPN installieren!\n"
"URL1: %s\n"
"URL2: %s"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:917
msgid "Invalid characters in '%s'"
msgstr "Ungültige Zeichen in '%s'"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:933
msgid "Choose your External Network Adapter!"
msgstr "Netzwerk Adapter auswählen!"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:951
msgid "No Network Adapter found!"
msgstr "Keine Netzwerk Adapter gefunden!"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:966
msgid "Please select your User-ID!"
msgstr "Bitte User-ID auswählen!"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:1013
msgid "Disconnect %s"
msgstr "Trennen %s"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:1017
msgid "Connect to %s"
msgstr "Verbinde zu %s"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:1025
msgid "Remove AutoConnect: %s"
msgstr "Deaktiviere automatische Verbindung: %s"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:1029
msgid "Set AutoConnect: %s"
msgstr "Aktiviere automatische Verbindung: %s"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:1044
msgid "[enabled]"
msgstr "[Ein]"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:1046
msgid "[disabled]"
msgstr "[Aus]"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:1047
msgid "Load Server Information %s"
msgstr "Lade Server Informationen %s"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:1060
msgid "Set Server-View Size [%sx%s]"
msgstr "Setze Server Fenster Größe [%sx%s]"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:1067
msgid "Update every: %s seconds"
msgstr "Aktualisiere alle: %s Sekunden"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:1085
msgid "Change DNS"
msgstr "DNS wechseln"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:1159
msgid "Set Primary DNS"
msgstr "Setze primären DNS"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:1169
msgid "Set Secondary DNS"
msgstr "Setze sekundären DNS"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:1331
msgid "Checking for Updates!"
msgstr "Nach Updates suchen!"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:1333
msgid "Downloading Configurations..."
msgstr "Lade Konfigurationen..."

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:1335
msgid "Requesting Certificates..."
msgstr "Zertifikate werden angefragt..."

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:1337
msgid "Please wait... Certificates requested from backend! (%s)"
msgstr "Bitte warten... Zertifikate werden vom Backend angefordert! (%s)"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:1339
msgid "Downloading Certificates..."
msgstr "Lade Zertifikate..."

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:1341
msgid "Extracting Configs and Certs..."
msgstr "Entpacke Konfigurationen und Zertifikate"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:1345
msgid "Disconnected! Have a nice and anonymous day!"
msgstr "Nicht verbunden! Hab einen schönen und anonymen Tag!"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:1359
msgid "Connecting to %s"
msgstr "Verbinde zu %s"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:1365
msgid "Testing connection to %s"
msgstr "Teste Verbindung zu %s"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:1370
msgid "Connection to %s unstable or failed!"
msgstr "Verbindung zu %s instabil oder fehlgeschlagen!"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:1384
msgid "Connected to %s [%s]:%s (%s) [ %s ] (%s / %s ms)"
msgstr "Verbunden zu %s [%s]:%s (%s) [ %s ] (%s / %s ms)"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:1547
msgid " [ disconnect ]"
msgstr " [ Trennen ]"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:1590
msgid "Close Servers"
msgstr "Server Fenster schließen"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:1593
msgid "Servers"
msgstr "Server Fenster"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:1603
msgid "Close Account"
msgstr "Konto Fenster schließen"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:1605
msgid "Account"
msgstr "Konto Fenster"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:1615
msgid "Close Settings"
msgstr "Einstellungen schließen"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:1617
msgid "Settings"
msgstr "Einstellungen"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:1630
msgid "About"
msgstr "Über"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:1638
msgid "Quit"
msgstr "Beenden"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:1697
msgid "Error: Update Timeout"
msgstr "Fehler: Update Timeout"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:1697
msgid ""
"Update took too long...aborted!\n"
"Please retry in few minutes..."
msgstr ""
"Aktualisierung dauerte zu lange und wurde abgebrochen! Bitte in wenigen "
"Minuten nochmal versuchen..."

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:1708
msgid "Certificates and Configs updated!"
msgstr "Zertifikate und Konfiguration erneuert!"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:1708
#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:1721
msgid "oVPN Update OK!"
msgstr "oVPN Update OK!"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:1711
msgid "Extraction failed!"
msgstr "Entpacken fehlgeschlagen!"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:1711
#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:1713
#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:1716
#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:1718
#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:1726
msgid "Error: def inThread_timer_check_certdl"
msgstr "Fehler: def inThread_timer_check_certdl"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:1713
msgid "Failed to download certificates"
msgstr "Fehler beim Runterladen der Zertifikate!"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:1716
msgid "Failed to request certificates!"
msgstr "Fehler beim Anfragen der Zertifikate!"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:1718
msgid "Failed to download configurations!"
msgstr "Fehler beim Runterladen der Konfigurationen!"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:1721
msgid "No update needed!"
msgstr "Kein Update nötig!"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:1726
msgid "Failed to check for updates!"
msgstr "Fehler beim Prüfen auf Updates!"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:1971
msgid "oVPN Server - %s"
msgstr "oVPN Server - %s"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:2043
msgid "oVPN Server [ %s ]"
msgstr "oVPN Server [ %s ]"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:2179
msgid "oVPN Account - %s"
msgstr "oVPN Konto - %s"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:2199
msgid "Settings -> Updates: 'Load Account Info' [disabled]"
msgstr "Einstellungen -> Updates: Lade Konto Informationen [AUS]"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:2201
msgid "No data loaded! Retry in few seconds..."
msgstr "Keine Daten geladen! Bitte in ein paar Sekunden erneut probieren..."

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:2298
msgid "oVPN Settings - %s"
msgstr "oVPN Einstellungen - %s"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:2322
msgid "Security Settings\n"
msgstr "Sicherheits Einstellungen\n"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:2331
msgid " Security "
msgstr " Sicherheit "

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:2339
msgid "Options\n"
msgstr "Einstellungen\n"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:2349
msgid " Options "
msgstr " Einstellungen "

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:2357
msgid "Updates\n"
msgstr "Updates\n"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:2362
msgid " Updates "
msgstr " Updates "

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:2372
msgid "Restore Firewall Backups\n"
msgstr "Firewall Backups wiederherstellen\n"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:2374
msgid " Backups "
msgstr " Sicherungen "

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:2382
msgid "Use Windows Firewall (default: ON) "
msgstr "Windows Firewall benutzen (Standard: AN) "

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:2412
msgid "TAP-Adapter block outbound (default: OFF)"
msgstr "TAP-Adapter ausgehend blocken (Standard: AUS)"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:2443
msgid "Block Internet on Disconnect or Quit (default: ON)"
msgstr "Internet sperren bei Trennung oder Beenden (Standard: AN)"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:2471
msgid "Disable FW question on Quit (default: OFF)"
msgstr "Firewall Abfrage beim beenden abschalten (Standard: AUS)"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:2499
msgid "Clear Rules on Connect (default: OFF)"
msgstr "Regeln beim Verbinden löschen (Standard: AUS)"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:2519
#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:2549
msgid "Could not export Windows Firewall Backup!"
msgstr "Konnte Windows Firewall Sicherung nicht exportieren!"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:2519
#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:2549
msgid "Error: Windows Firewall Backup failed"
msgstr "Fehler: Windows Firewall Sicherung fehlgeschlagen"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:2529
msgid "Backup on Start / Restore on Quit (default: OFF)"
msgstr "Sicherung/Wiederherstellung bei Start und Beenden (Standard: AUS)"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:2559
msgid "DNS Leak Protection (default: ON)"
msgstr "DNS Leck Schutz (Standard: AN)"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:2589
msgid "Disable '%s' on Disconnect (default: OFF)"
msgstr "Deaktiviere '%s' bei Trennung (Standard: AUS)"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:2627
msgid "Update Configs on Start (default: OFF)"
msgstr "Beim Start auf Konfig Update prüfen (Standard: AUS)"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:2652
msgid "Load Account Info (default: OFF)"
msgstr "Lade Konto Informationen (Standard: AUS)"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:2686
msgid "Load Server Info (default: OFF)"
msgstr "Lade Server Informationen (Standard: AUS)"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:2721
msgid "Debug Mode (default: OFF)"
msgstr "Debug Modus (Standard: AUS)"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:2737
msgid "Debug Mode Enabled"
msgstr "Debug Modus eingeschaltet"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:2737
msgid ""
"Logfile:\n"
"'%s'"
msgstr ""
"Log Datei:\n"
"'%s'"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:2746
msgid "Select Network Adapter"
msgstr "Netzwerk Adapter wählen"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:2757
msgid "Use IPv4 Entry Server with Exit to IPv4 (standard)"
msgstr "Benutze IPv4 Eintrittsserver mit Ausgang nach IPv4 (Standard)"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:2760
msgid "Use IPv4 Entry Server with Exits to IPv4 + IPv6"
msgstr "Benutze IPv4 Eintrittsserver mit Ausgang nach IPv4 + IPv6"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:2785
msgid "Change App Theme (default: ms-windows)"
msgstr "App Thema wählen (Standard: MS-Windows)"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:2818
msgid "Change App Font Size (default: 9)"
msgstr "App Schriftgröße wählen (Standard: 9)"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:2864
msgid "Change Language"
msgstr "Sprache auswählen"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:2900
msgid "Disable About and Quit on Connection (default: ON)"
msgstr "Blende Über und Beenden aus bei Verbindung (Standard: AN)"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:2921
msgid "Normal Config Update"
msgstr "Normales Konfigurations Update"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:2930
msgid "Forced Config Update"
msgstr "Erzwungenes Konfigurations Update"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:2939
msgid "Reset API-Login"
msgstr "API-Login zurücksetzen"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:3173
msgid "Please wait!"
msgstr "Bitte warten!"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:3173
msgid "Update is running."
msgstr "Aktualisierung läuft."

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:3249
msgid "Error: Certs not found!"
msgstr "Fehler: Zertifikate nicht gefunden!"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:3249
msgid "Files missing: '%s'"
msgstr "Fehlende Dateien: '%s'"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:3277
msgid "Could not start Windows Firewall!"
msgstr "Konnte die Windows Firewall nicht starten!"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:3277
msgid "Error: def inThread_spawn_openvpn_process"
msgstr "Fehler: def inThread_spawn_openvpn_process"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:3588
msgid "Error: DNS restore 2nd"
msgstr "Fehler: DNS Wiederherstellung 2."

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:3588
msgid "Error: Restore Secondary DNS to %s failed."
msgstr "Fehler: Sekundären DNS wiederherstellen zu %s fehlgeschlagen."

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:3591
msgid "Error: DNS restore 1st"
msgstr "Fehler: DNS Wiederherstellung 1."

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:3591
msgid "Error: Restore Primary DNS to %s failed."
msgstr "Fehler: Primären DNS wiederherstellen zu %s fehlgeschlagen."

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:3654
msgid "def win_netsh_read_dns_to_backup: failed!"
msgstr "def win_netsh_read_dns_to_backup: fehlgeschlagen!"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:3687
msgid ""
"Error:\n"
"SSL Root Certificate for %s not loaded %s"
msgstr ""
"Fehler:\n"
"SSL Root Zertifikat für %s nicht geladen %s"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:3687
msgid "Error: SSL CA Cert #1"
msgstr "Fehler: SSL CA Cert #1"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:3690
msgid ""
"Error:\n"
"Invalid SSL Root Certificate for %s in:\n"
"'%s'\n"
"hash is:\n"
"'%s'\n"
"but should be '%s'"
msgstr ""
"Fehler:\n"
"Ungültiges root Zertifikat für %s in:\n"
"'%s'\n"
"hash ist:\n"
"'%s'\n"
"sollte aber '%s'"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:3690
msgid "Error: SSL CA Cert #2"
msgstr "Fehler: SSL CA Cert #2"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:3693
msgid ""
"Error:\n"
"SSL Root Certificate for %s not found in:\n"
"'%s'!"
msgstr ""
"Fehler:\n"
"root Zertifikat für %s nicht gefunden in:\n"
"'%s'!"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:3693
msgid "Error: SSL CA Cert #3"
msgstr "Fehler: SSL CA Cert #3"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:3882
#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:3906
msgid "Error: netsh.exe not found!"
msgstr "Fehler: netsh.exe nicht gefunden!"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:3921
#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:3945
msgid "Error: route.exe not found!"
msgstr "Fehler: route.exe nicht gefunden!"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:3961
#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:3973
msgid "ipconfig.exe not found!"
msgstr "ipconfig.exe nicht gefunden!"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:4008
msgid "oVPN.to Setup"
msgstr "oVPN.to Setup"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:4009
msgid "Enter your oVPN.to Details"
msgstr "oVPN.to Daten eingeben:"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:4018
msgid "User-ID:"
msgstr "User-ID:"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:4027
msgid "API-Key:"
msgstr "API-Key:"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:4131
#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:4141
msgid "Error: Update failed"
msgstr "Fehler: Update fehlgeschlagen"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:4184
msgid "Server Window Size"
msgstr "Server Fenster Größe"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:4188
msgid "Width (pixel):"
msgstr "Breite (pixel):"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:4192
msgid "Height (pixel):"
msgstr "Höhe (pixel):"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:4196
msgid ""
"Enter width and height\n"
"\n"
"Leave blank for default"
msgstr ""
"Breite und Höhe eingeben\n"
"\n"
"Leer lassen für Standard Werte"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:4243
msgid "Load Data every X seconds"
msgstr "Lade Daten alle X Sekunden"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:4247
msgid "seconds:"
msgstr "Sekunden:"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:4294
msgid "Switched to IPv4+6"
msgstr "Umgeschaltet auf IPv4+6"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:4294
#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:4311
msgid ""
"Changed Option:\n"
"\n"
"Use 'Forced Config Update' to get new configs!\n"
"\n"
"You have to join 'IPv6 Beta' on https://%s to use any IPv6 options!"
msgstr ""
"Geänderte Konfiguration:\n"
"\n"
"Benutze 'Erzwungenes Konfigurations Update'  um neue Konfigs zu erhalten!\n"
"\n"
"Du musst der 'IPv6 Beta' beitreten auf https://%s um IPv6 nutzen zu können!"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:4311
msgid "Switched to IPv6+4"
msgstr "Umgeschaltet zu IPv6+4"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:4361
msgid "No API-Key!"
msgstr "Kein API-Key!"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:4361
#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:4392
#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:4403
#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:4415
msgid "Error: def API_REQUEST"
msgstr "Fehler: def API_REQUEST"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:4392
#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:4415
#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:4696
#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:4753
msgid "Invalid User-ID or API-Key or Account expired!"
msgstr "Ungültige User-ID oder API-Key oder Zugang abgelaufen!"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:4403
msgid "API requests on: %s failed!"
msgstr "API Anfrage an: %s fehlgeschlagen!"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:4516
msgid "Error: def move_appdata"
msgstr "Fehler: def move_appdata"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:4516
msgid "Move Userdata to new folder: '%s'"
msgstr "Verschiebe Benutzerdaten in neues Verzeichnis: '%s'"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:4552
msgid "Could not read Servers Remote-IP:Port from config: %s"
msgstr "Konnte Servers Remote-IP:Port nicht aus der Konfig auslesen: %s"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:4561
msgid "Could not read Servers Protocol from config: %s"
msgstr "Konnte Servers Protocol nicht aus der Konfig auslesen: %s"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:4573
msgid "Could not read Servers Cipher from config: %s"
msgstr "Konnte Servers Cipher nicht aus der Konfig auslesen: %s"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:4599
msgid "Error: def load_ovpn_server"
msgstr "Fehler: def load_ovpn_server"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:4599
msgid "Load openVPN Server failed"
msgstr "openVPN Server Start fehlgeschlage"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:4775
msgid "Platform '%s' not supported"
msgstr "Platform '%s' nicht unterstützt"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:4775
#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:4826
#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:4830
#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:4914
#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:4967
#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:4970
msgid "Error!"
msgstr "Fehler!"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:4791
msgid ""
"openVPN Setup downloaded and hash verified OK!\n"
"\n"
"Please start setup from file:\n"
"'%s'\n"
"\n"
"Verify GPG with:\n"
"'%s'"
msgstr ""
"openVPN Setup heruntergeladen! Hash Check: OK!\n"
"\n"
"Bitte mit folgender Datei installieren:\n"
"'%s'\n"
"\n"
"GPG überprüfen mit:\n"
"'%s'"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:4793
msgid ""
"openVPN Setup downloaded but hash verify failed!\n"
"Please install openVPN!\n"
"URL1: %s\n"
"URL2: %s"
msgstr ""
"openVPN Setup heruntergeladen, aber hash Überprüfung fehlgeschlagen!\n"
"Bitte installiere openVPN manuell!\n"
"URL1: %s\n"
"URL2: %s"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:4800
msgid ""
"Install OpenVPN %s (%s) (%s)\n"
"\n"
"Starting download (~1.8 MB) from:\n"
"'%s'\n"
"to\n"
"'%s'\n"
"\n"
"Please wait..."
msgstr ""
"Installiere OpenVPN %s (%s) (%s)\n"
"\n"
"Starte download (~1.8 MB) von:\n"
"'%s'\n"
"nach\n"
"'%s'\n"
"\n"
"Bitte warten..."

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:4800
#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:4866
msgid "Setup: openVPN"
msgstr "Setup: openVPN"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:4826
msgid ""
"Invalid File hash: %s !\n"
"localhash = '%s'\n"
"but should be = '%s'"
msgstr ""
"Ungültiger Datei hash: %s !\n"
"localhash = '%s'\n"
"sollte sein = '%s'"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:4830
msgid "Failed remove file: %s"
msgstr "Datei entfernen fehlgeschlagen: %s"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:4866
msgid ""
"OpenVPN not found!\n"
"\n"
"Please select openvpn.exe on next window!\n"
"\n"
"If you did not install openVPN yet: click cancel on next window!"
msgstr ""
"OpenVPN nicht gefunden!\n"
"\n"
"Bitte openvpn.exe im nächsten Fenster auswählen!\n"
"\n"
"Wenn openVPN noch nicht installiert ist im nächsten Fenster auf abbrechen "
"klicken!"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:4867
msgid "Select openvpn.exe or Cancel to install openVPN"
msgstr "openvpn.exe auswählen oder Abbrechen um openVPN zu installieren"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:4914
msgid ""
"WARNING! Failed to verify files in\n"
"'%s'\n"
"\n"
"Uninstall openVPN and restart oVPN Client Software!\n"
"\n"
"Or install openVPN from URL:\n"
"%s[debug self.LAST_FAILED_CHECKFILE = '%s']"
msgstr ""
"WARNUNG! Dateien konnten nicht verifiziert werden in\n"
"'%s'\n"
"\n"
"openVPN deinstallieren und oVPN Client neu starten!\n"
"\n"
"Alternativ openVPN manuell installieren:\n"
"%s[debug self.LAST_FAILED_CHECKFILE = '%s']"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:4919
msgid "Could not detect openVPN Version!"
msgstr "Konnte openVPN Version nicht ermitteln!"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:4943
msgid "Could not find openVPN"
msgstr "Konnte openVPN nicht finden"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:4967
msgid "Invalid Hash: '%s'! is '%s' != '%s'"
msgstr "Ungültiger Hash: '%s'! is '%s' != '%s'"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:4970
msgid "Invalid content '%s' in '%s'"
msgstr "Ungültiger Inhalt '%s' in '%s'"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:4987
msgid "arch '%s' not supported!"
msgstr "Struktur '%s' nicht unterstützt!"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:5195
msgid "Quit oVPN.to Client"
msgstr "oVPN.to beenden"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:5199
msgid "Do you really want to quit?"
msgstr "Wirklich beenden?"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:5265
msgid "Firewall Settings"
msgstr "Firewall Einstellungen"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:5269
msgid ""
"Restore previous firewall settings?\n"
"\n"
"Press 'YES' to restore your previous firewall settings!\n"
"Press 'NO' to set profiles to 'blockinbound,blockoutbound'!"
msgstr ""
"Vorherige Firewall Einstellungen wiederherstellen?\n"
"\n"
"JA, vorherige wiederherstellen!\n"
"NEIN, Setze 'blockinbound,blockoutbound'!"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:5272
msgid ""
"Allow outgoing connection to internet?\n"
"\n"
"Press 'YES' to set profiles to 'blockinbound,allowoutbound'!\n"
"Press 'NO' to set profiles to 'blockinbound,blockoutbound'!"
msgstr ""
"Ausgehende Verbindungen erlauben?\n"
"\n"
"JA, setze 'blockinbound,allowoutbound'!\n"
"NEIN, setze 'blockinbound,blockoutbound'!"

#: E:\Persoenlich\ovpn-client\.\ovpn_client.py:5327
msgid "Error"
msgstr "Fehler"

#~ msgid "Change App Theme"
#~ msgstr "App Thema wechseln"

#~ msgid "No Firewall Backups found!"
#~ msgstr "Keine Firewall Sicherungen gefunden!"

#~ msgid "Cannot restore Backups when use Windows Firewall is off!"
#~ msgstr ""
#~ "Wiederherstellung nicht möglich wenn Nutzung der Firewall deaktiviert ist!"

#~ msgid "Cannot restore Backups when oVPN is connected!"
#~ msgstr "Wiederherstellung nicht möglich solange oVPN verbunden ist!"

#~ msgid "Firewall"
#~ msgstr "Firewall"

#~ msgid "Restore Firewall Backups"
#~ msgstr "Firewall Einstellungen wiederherstellen"

#~ msgid "Select: IPv4 Entry Server with Exit to IPv4 (standard)"
#~ msgstr "Auswählen: IPv4 Eintritt Server mit Ausgang nach IPv4 (Standard)"

#~ msgid "Select: IPv4 Entry Server with Exits to IPv4 + IPv6"
#~ msgstr "Auswählen: IPv4 Eintritts Server mit Ausgang nach IPv4 + IPv6"

#~ msgid "Options"
#~ msgstr "Optionen"

#~ msgid "IPv6 Options"
#~ msgstr "IPv6 Optionen"

#~ msgid "Disable FW question on Quit (default: ON)"
#~ msgstr "Firewall Abfrage beim Beenden abschalten (standard: AN)"

#~ msgid "Updates"
#~ msgstr "Updates"

#~ msgid "Reset API Login"
#~ msgstr "API Login zurücksetzen"

#~ msgid "oVPN Server Update"
#~ msgstr "oVPN Server Update"

#~ msgid "API-Login failed! Invalid User-ID or API-Key or Account expired!"
#~ msgstr "Ungültige User-ID oder API-Key oder Zugang abgelaufen!"

#~ msgid "Error: def load_accinfo_from_remote"
#~ msgstr "Error: def load_accinfo_from_remote"

#~ msgid "Clear Passphrase from RAM"
#~ msgstr "Passwort aus dem Speicher löschen"

#~ msgid "Clear Passphrase from CFG"
#~ msgstr "Passwort aus der Konfiguration löschen"

#~ msgid "oVPN Update failed"
#~ msgstr "oVPN Update fehlgeschlagen"

#~ msgid "Enter your Passphrase"
#~ msgstr "Passwort eingeben"

#~ msgid "Save Passphrase in File?"
#~ msgstr "Passwort in Datei speichern?"

#~ msgid "Passphrase:"
#~ msgstr "Passwort:"

#~ msgid ""
#~ "\n"
#~ "\n"
#~ "Enter a secure passphrase to encrypt your API-Login!"
#~ msgstr ""
#~ "\n"
#~ "\n"
#~ "Eine sicheres Passwort eingeben um den API-Login zu verschlüsseln!"

#~ msgid "Repeat:"
#~ msgstr "Wiederholen:"

#~ msgid "Invalid User-ID / API-Key or Account expired."
#~ msgstr "Ungültige(r) User-ID / API-Key oder Account abgelaufen."

#~ msgid "Primary DNS '%s' @ %s"
#~ msgstr "Primärer DNS '%s' @ %s"

#~ msgid "Secondary DNS '%s' @ %s"
#~ msgstr "Sekundärer DNS '%s' @ %s"

#~ msgid "Primary DNS: %s (%s)"
#~ msgstr "Primärer DNS: %s (%s)"

#~ msgid "Secondary DNS: %s (%s)"
#~ msgstr "Sekundärer DNS: %s (%s)"

#~ msgid "Load extended Server-View %s"
#~ msgstr "Lade Server Informationen %s"

#~ msgid "Change Theme"
#~ msgstr "App Thema wechseln"

#~ msgid "Choose App Theme"
#~ msgstr "App-Thema auswählen"

#~ msgid "DEBUG Mode %s"
#~ msgstr "DEBUG Modus %s"

#~ msgid "Set App Theme [%s]"
#~ msgstr "Setze App Thema [%s]"

#~ msgid "Windows Firewall Settings\n"
#~ msgstr "Windows Firewall Einstellungen\n"

#~ msgid " Firewall "
#~ msgstr " Firewall "

#~ msgid "Network Adapter Settings\n"
#~ msgstr "Netzwerk Adapter Einstellungen\n"

#~ msgid " Network "
#~ msgstr " Netzwerk "

#~ msgid "Update Settings\n"
#~ msgstr "Update Einstellungen \n"

#~ msgid "Update every: %s sec"
#~ msgstr "Aktualisiere jede: %s Sekunden"

#~ msgid "Refresh Window"
#~ msgstr "Fenster aktualisieren"

#~ msgid "Systray Menu -> Updates -> 'Load Account Info' [disabled]"
#~ msgstr "Tray Menü -> Updates -> 'Lade Konto Informationen' [Aus]"

#~ msgid "Load Account Info %s"
#~ msgstr "Lade Konto Informationen %s"

#~ msgid "def check_config_folders: failed"
#~ msgstr "def check_config_folders: fehlgeschlagen"

#~ msgid "def read_options_file: failed!"
#~ msgstr "def read_options_file: Fehlgeschlagen!"

#~ msgid "Invalid characters in self.WIN_TAP_DEVICE = '%s'"
#~ msgstr "Ungültige Zeichen in self.WIN_TAP_DEVICE = '%s'"

#~ msgid "def inThread_timer_check_certdl: extraction failed"
#~ msgstr "def inThread_timer_check_certdl: Extraktion fehlgeschlagen"

#~ msgid "Error: oVPN Update failed!"
#~ msgstr "Fehler: oVPN Update fehlgeschlagen!"

#~ msgid "def inThread_timer_check_certdl: downloading certs failed"
#~ msgstr "def inThread_timer_check_certdl: Zertifikat download fehlgeschlagen"

#~ msgid ""
#~ "def inThread_timer_check_certdl: self.API_REQUEST(API_ACTION = "
#~ "requestcerts): failed"
#~ msgstr ""
#~ "def inThread_timer_check_certdl: self.API_REQUEST(API_ACTION = "
#~ "requestcerts): fehlgeschlagen"

#~ msgid ""
#~ "def inThread_timer_check_certdl: self.API_REQUEST(API_ACTION = "
#~ "getconfigs): failed"
#~ msgstr ""
#~ "def inThread_timer_check_certdl: self.API_REQUEST(API_ACTION = "
#~ "getconfigs): fehlgeschlagen"

#~ msgid ""
#~ "def inThread_timer_check_certdl: self.API_REQUEST(API_ACTION = "
#~ "lastupdate): failed"
#~ msgstr ""
#~ "def inThread_timer_check_certdl: self.API_REQUEST(API_ACTION = "
#~ "lastupdate): fehlgeschlagen"

#~ msgid "def inThread_timer_check_certdl: failed"
#~ msgstr "def inThread_timer_check_certdl: fehlgeschlagen"

#~ msgid "Files missing: '%s' % (self.ovpn_server_dir)"
#~ msgstr "Fehlende Datei: '%s' % (self.ovpn_server_dir)"

#~ msgid ""
#~ "def inThread_spawn_openvpn_process: Could not start Windows Firewall!"
#~ msgstr ""
#~ "def inThread_spawn_openvpn_process: Windows Firewall konnte nicht "
#~ "gestartet werden!"

#~ msgid "def inThread_spawn_openvpn_process: self.inThread_timer_ovpn_ping"
#~ msgstr "def inThread_spawn_openvpn_process: self.inThread_timer_ovpn_ping"

#~ msgid "def win_ipconfig_flushdns: ipconfig.exe not found!"
#~ msgstr "def win_ipconfig_flushdns: ipconfig.exe not found!"

#~ msgid "def win_ipconfig_displaydns: ipconfig.exe not found!"
#~ msgstr "def win_ipconfig_displaydns: ipconfig.exe not found!"

#~ msgid "Error: API-Request"
#~ msgstr "Fehler: API-Request"

#~ msgid "def API_REQUEST: requests error on: %s failed!"
#~ msgstr "def API_REQUEST: requests error on: %s fehlgeschlagen!"

#~ msgid "def API_REQUEST: self.curldata = '%s'"
#~ msgstr "def API_REQUEST: self.curldata = '%s'"

#~ msgid "Move Userdata to new folder!"
#~ msgstr "Verschiebe Nutzer Daten in neues Verzeichnis!"

#~ msgid "def move_appdata: '%s'"
#~ msgstr "def move_appdata: '%s'"

#~ msgid "def load_ovpn_server: failed"
#~ msgstr "def load_ovpn_server: fehlgeschlagen"

#~ msgid "def win_detect_openvpn: failed"
#~ msgstr "def win_detect_openvpn: fehlgeschlagen"

#~ msgid "Error: WINDIR '%s' not found!"
#~ msgstr "Fehler: WINDIR '%s' nicht gefunden!"

#~ msgid "Error: self.WIN_NETSH_EXE '%s' not found!"
#~ msgstr "Fehler: self.WIN_NETSH_EXE '%s' nicht gefunden!"

#~ msgid "Error: self.WIN_ROUTE_EXE '%s' not found!"
#~ msgstr "Fehler: self.WIN_ROUTE_EXE '%s' nicht gefunden!"

#~ msgid "Error: self.WIN_IPCONFIG_EXE '%s' not found!"
#~ msgstr "Fehler: self.WIN_IPCONFIG_EXE '%s' nicht gefunden!"

#~ msgid "Error: self.WIN_TASKKILL_EXE '%s' not found!"
#~ msgstr "Fehler: self.WIN_TASKKILL_EXE '%s' nicht gefunden!"

#~ msgid "def check_winapp_dir could not create app_dir: %s"
#~ msgstr "def check_winapp_dir konnte App Verzeichnis nicht anlegen: %s"
